src/test/java/com/epam/machine/controller/RegisterServletTest.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.machine.controller;
+
+import org.junit.Test;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.mockito.Mockito.*;
```

Comment:
Do not use wildcards in import section, adjust setting in IDE to not use *. Settings->Editor->Code Style->Java-> Imports